manifests/pipecd/values.yaml

```diff
@@ -211,6 +214,56 @@ prometheus:
               action: keep
               regex: admin
 
+        # Scrape config for API servers.
+        - job_name: 'kubernetes-apiservers'
```

Comment:
nit: Remove `'` to be unified with other jobs.

Reply:
for sure